code/modules/mob/living/carbon/human/species/vox.dm

```diff
@@ -102,6 +102,7 @@
 	..()
 	updatespeciescolor(H)
 	H.update_icons()
+	ADD_TRAIT(H, TRAIT_SKITTISH, "species")
```

Comment:
This is exactly the kind of thing we're trying to avoid traits becoming, which is why I've largely held back on altering traits *too* much at present.

Traits are best handled to be as broad and generalized as possible, and for something like species...them being a list as opposed to snowflaked `ADD_TRAIT` everywhere.

Reply:
I see, I was simply looking how the Waterbreathing trait which was recently added was applied, and copied the way that was handled. Where's a good place to handle this?